services/logs/filter_test.go

```diff
@@ -9,10 +9,11 @@ import (
 	gethTypes "github.com/onflow/go-ethereum/core/types"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
+	"github.com/syndtr/goleveldb/leveldb/errors"
```

Comment:
Where did that come from? :thinking: Probably wrong import?

Reply:
yep

Reply:
tnx